include/OpenColorIO/OpenColorIO.h

```diff
@@ -1033,9 +1028,9 @@ class OCIOEXPORT Config
     //
 
     // TODO: Move to .rst
-    // Create a :cpp:class:`Processor` to assemble a transformation between two
-    // color spaces.  It may then be used to create a :cpp:class:`CPUProcessor`
-    // or :cpp:class:`GPUProcessor` to process/convert pixels.
+    // Create a Processor to assemble a transformation between two
+    // color spaces.  It may then be used to create a CPUProcessor
+    // or GPUProcessor to process/convert pixels.
     // rst:: Get the processor to apply a ColorSpaceTransform from a source to a destination
```

Comment:
Is the "rst::" still needed?

Reply:
No sir. 

There are some regions that differentiate between documentation directly associated with class and function definitions, and documentation that more generally explain concepts. I'm not 100% certain of this, but it seems like doxygen only knows what to do with the former, and I think I may have left some `rst::` bits in place either because I hadn't figured out what to do with the latter, or because I missed them. Or both.

In other words, if the comments aren't directly above a definition of some sort, they seem to get ignored when rendering. For instance, there might be a block of comments that introduces a concept worth documenting that applies to a subset of class functions. This makes sense visually and organizationally in the header itself, but I'm finding that these "floating comments" just get ignored by doxygen. In some cases, I've moved these comment blocks up to the class-level documentation so they get rendered, but I'm not sure that's the best solution either, since it undermines the author's original intention, in terms of making the header file itself more readable, which is of the upmost importance. On the other hand, it's not intuitive to authors that these comments will not get rendered at all. 

I'm not sure what the best solution is. I think maybe these comments should stay in place, as the author intends; but worthwhile concepts should be manually copied or promoted elsewhere for rendering. Considering how many gems are hidden in the comments in much of the cpp code, I wonder if its worth stubbing out separate hand-authored "concept" pages. 

(From a user perspective, I'd kind of expect the API docs to supplement Concept docs curated elsewhere)

Reply:
This wasn't as large of an issue as I thought. I rearranged a couple of notes in order to make sure they were rendered to HTML, but in general, I erred on the side of developer-facing in-line readability.